apps/prairielearn/src/sync/fromDisk/assessments.js

```diff
@@ -119,16 +119,22 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
 
   let alternativeGroupNumber = 0;
   let assessmentQuestionNumber = 0;
+  let assessmentCanView = _.has(assessment, 'canView') ? assessment?.canView : [];
```

Comment:
Is there a reason you're using lodash's `has` instead of optional chaining and nullish coalescing like `canView`/`canSubmit` used before this PR?

Reply:
I pattern matched against some of the existing code that accessed object properties.. but if it's preferred, should I switch back to optional chaining and nullish coalescing? Should be no problem!

Reply:
Yeah, this syncing stuff has some of the oldest lines of code in all of PL; the usage of lodash here predates optional chaining and nullish coalescing by many years. I'd consider using native syntax to be the more idiomatic way to do this now!